kotlinx-coroutines-core/common/src/Supervisor.kt

```diff
@@ -42,11 +42,15 @@ public fun SupervisorJob0(parent: Job? = null) : Job = SupervisorJob(parent)
  * Creates a [CoroutineScope] with [SupervisorJob] and calls the specified suspend block with this scope.
  * The provided scope inherits its [coroutineContext][CoroutineScope.coroutineContext] from the outer scope, but overrides
  * context's [Job] with [SupervisorJob].
+ * This function returns as soon as the given block and all its children coroutines are completed.
```

Comment:
https://english.stackexchange.com/questions/468019/children-as-adjective-as-opposed-to-child
```suggestion
 * This function returns as soon as the given block and all its child coroutines are completed.
```